doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,71 +210,134 @@ infrastructure itself from the workloads compute resources.
+- **External / Application Load Balancing**: As Kubernetes Ingress, Egress and
+Services have no support for all the protocols needed in telecommunication
+environments (Diameter, SIP, LDAP, etc) and their capacity is limited, the
+architecture includes the use of alternative load balancers, including external
+or ones built into the application. Management of external load balancers must
+be possible via Kubernetes API objects.
+- **Other Features**: these additional features that are required by the
+networking solution as a whole, may be delivered by the **"Default CNI Plugin"**,
+or the **"CNI multiplexer/meta-plugin"** if it is deployed. For example,
+the integration of SDN solutions required by `req.inf.ntw.05` is enabled
+via CNI integration.
+- **Service Mesh**: The well known service meshes are "application service meshes"
+that address and interact with the application layer 7 protocols (eg.: HTTP)
+only. Therefore, their support is not required in this architecture, as these
+service meshes are outside the scope of the infrastructure layer of this
+architecture.
+
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
+
+<!--The above diagram is maintained here:
+https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
+
+There are several types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
+| Requirement | Networking Solution with Multus | Networking Solution with DANM | Networking Solution with Tungsten Fabric | Networking Solution with NSM |
+|---|---|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | Supported | TBC |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported | Supported | TBC |
+| Network Policies (Security Groups) (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> | Supported | Supported via the default CNI plugin |
```

Comment:
We should add a note, to the default CNI pluign part that several CNI plugins do not support Network Policies.

Reply:
yes good point - have clarified in https://github.com/cntt-n/CNTT/pull/2121/commits/f31fff76dfac38718ce808e2781d1e6074a16b80

Reply:
@CsatariGergely are you able to resolve this please?